apps/ledger-live-mobile/e2e/models/onboarding/onboardingSteps.js

```diff
@@ -10,9 +10,16 @@ export default class OnboardingSteps {
     await testHelpers.tapByText("Get started");
   }
 
+  static async DoIOwnDevice(answer = true) {
+    answer
+      ? await testHelpers.tapByText("Yes, I do")
```

Comment:
not a fan of this method of selecting elements. isnt there an id you could target instead?

Reply:
don't have a preference because i see pro & cons in approaches. maybe it's valuable to have kinda both approach?

pros: we can really assert presence of a test a from user perspective + we don't need to maintain id
cons: it's something to maintain if we change the wording

Reply:
Yeah exactly that. It's tested from a user's point of view and also a lot quicker to implement if there's text since you don't have to go trawling through the components. However it's a bit less maintainable if the wording is changed. I'd say we keep it for now and see how much maintenance it takes